tests/unit/fake_storage.py

```diff
@@ -0,0 +1,63 @@
+from sem_covid.adapters.abstract_store import *
```

Comment:
their implementations can be filled in with what is currently available in the unit/conftest.py